networkmanager_dmenu

```diff
@@ -287,30 +290,43 @@ def get_selection(client, aps, vpns, gsms, others):
                  gsms: list of Actions
                  others: list of Actions
     """
-    formated_aps = [str(ap) for ap in aps]
-    formated_vpns = [str(vpn) for vpn in vpns]
-    formated_gsms = [str(gsm) for gsm in gsms]
-    formated_others = [str(other) for other in others]
+
+    conf = configparser.ConfigParser()
+    conf.read(expanduser("~/.config/networkmanager-dmenu/config.ini"))
+
+    try:
+        rofi_highlight = conf.getboolean('dmenu', 'rofi_highlight')
+    except (configparser.NoOptionError, configparser.NoSectionError):
+        rofi_highlight = False
 
     inp = []
-    if formated_aps:
-        inp += formated_aps + [""]
-    if vpns:
-        inp += formated_vpns + [""]
-    if gsms:
-        inp += formated_gsms + [""]
-    inp += formated_others
+    empty_action = [Action('', None)]
+    all_actions = []
+    all_actions += aps + empty_action if aps else []
+    all_actions += vpns + empty_action if vpns else []
+    all_actions += gsms + empty_action if gsms else []
+    all_actions += others
+
+    if rofi_highlight:
+        inp = [str(action) for action in all_actions]
+    else:
+        inp = [('** ' if action.is_active else '   ') + str(action) for action in all_actions]
+    active_lines = [index for index, action in enumerate(all_actions) if action.is_active]
 
     inp_bytes = "\n".join([str(i) for i in inp]).encode(ENC)
-    sel = Popen(dmenu_cmd(len(inp)), stdin=PIPE, stdout=PIPE,
+    sel = Popen(dmenu_cmd(len(inp), active_lines=active_lines), stdin=PIPE, stdout=PIPE,
                 env=ENV).communicate(input=inp_bytes)[0].decode(ENC)
 
     if not sel.rstrip():
         sys.exit()
 
-    action = [i for i in aps + vpns + gsms + others
-              if str(i) == sel.rstrip("\n")]
-    assert len(action) == 1, "Selection was ambiguous: {}".format(str(sel))
+    if not rofi_highlight:
+        action = [i for i in aps + vpns + gsms + others
+                  if prepend_marker_to_active_action(i) == str(sel.rstrip("\n"))]
```

Comment:
The variable `prepend_marker_to_active_action` doesn't seem to exist anywhere else and the script will fail if rofi_highlight is defined as `False`.